docs/converters/converter.md

```diff
@@ -0,0 +1,50 @@
+<!--
+SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dynamic.grid.calculation@alliander.com>
+
+SPDX-License-Identifier: MPL-2.0
+-->
+
+# Converters
+
+There are 4 types of converters present as of now. PGM JSON Converter, VisonExcelConverter, GaiaExcelConverter and Pandapower Converter.
+Among them, VisonExcelConverter and GaiaExcelConverter are [tabular converters](tabular_converter.md) for excel exports of vision and gaia respectively.
+
+PGM JSON converter converts to and from the power-grid-model JSON data format. 
+This json format makes it convenient to evaluate the grid data in perspective of input to power-grid-model. 
+More details about the JSON format are mentioned in {external:py:class}`power_grid_model.utils` and example of it is in [Make Test Dataset](power-grid-model:exapmles/Make%20Test%20Dataset.ipynb) in power-grid-model repository.
```

Comment:
Should {external:py:class} be visible in de markdown? I guess something goes wrong here ;)

Reply:
It is needed for generation of read the docs.

Reply:
Hmm okay, I don't like that it is showing in the rendered markdown, but if this is the way to go with readthedocs then I think it's okay

Reply:
Yes, things like these are visible in all of the docs of both pgm and pgm-io.  No way around it. We should now always try to redirect users to the readthedocs site as there is reading via markdowns is not as fluid as it should be. Other libraries do the same as well.

Reply:
Thanks